addons/hr_skills/views/hr_views.xml

```diff
@@ -282,7 +282,7 @@
                                 <span class="ps-0" style="flex:1">
                                     <field name="skill_level_id"
                                             attrs="{'readonly': [('skill_id', '=', False)]}"
-                                            context="{'from_skill_level_dropdown': True}" />
+                                            context="{'from_skill_level_dropdown': True, 'skill_type_id': skill_type_id}" />
```

Comment:
If you use `default_<field name>` it will be automatically picked-up by default_get and it's not needed to do anything in create() :-D

```suggestion
                                            context="{'from_skill_level_dropdown': True, 'default_skill_type_id': skill_type_id}" />
```